packs/azure/actions/lib/formatters.py

```diff
@@ -0,0 +1,16 @@
+__all__ = [
+    'to_node_dict'
+]
+
+
+def to_node_dict(node):
```

Comment:
Can pip install dependencies from a file location? If so, I like the requirements approach.